Assets/MixedRealityToolkit/Utilities/ClippingPrimitive.cs

```diff
@@ -93,6 +130,20 @@ protected void OnDisable()
         {
             UpdateRenderers();
             ToggleClippingFeature(false);
+
+            if (cameraMethods != null)
+            {
+                cameraMethods.OnCameraPreRender -= OnCameraPreRender;
```

Comment:
I noted we unsubscribed properly here in disable, but what about OnEnable?
Does this fail after being disabled once?

Reply:
Okay, had a conversation with @Cameron-Micka about this topic. I didn't know `Start` isn't called when an object is re-enabled. I'm shocked I haven't run into problems yet. :) Good catch - I'll update.